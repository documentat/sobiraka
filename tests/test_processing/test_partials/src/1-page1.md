# Two includes

{% include 'part.md' %}

{% include 'part.md' %}
