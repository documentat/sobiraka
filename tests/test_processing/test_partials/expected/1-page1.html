<h1>Two includes</h1>

<p>This text is from a partial.</p>
<p>This text is from a partial.</p>
